source/crud/tests/v1/write/replaceOne-collation.json

```diff
@@ -10,6 +10,7 @@
     }
   ],
   "minServerVersion": "3.4",
+  "serverless": "forbid",
```

Comment:
Same behavior with the operation being an individual request of `bulkWrite` operation.